sponsorship_compassion/models/contract_group.py

```diff
@@ -67,7 +67,7 @@ def build_inv_line_data(
     def _get_partner_for_contract(self, contract, gift_wizard=False):
         if gift_wizard and contract.send_gifts_to:
             return contract[contract.send_gifts_to]
-        return super()._get_partner_for_contract(contract, gift_wizard)
+        return super()._get_partner_for_contract(contract)
```

Comment:
What does this change stands for?

Reply:
I don't think it's a valid change, or what was your intent here?

Reply:
It was to fix some unit test, but it's indeed a mistake (I forgot to upgrade another module when running the test). I'll revert this change